src/api/comments.ts

```diff
@@ -0,0 +1,10 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
```

Comment:
Consider adding error handling for the `getComments` API call. This can be done by chaining a `.catch` block to handle potential errors during the request.